packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -146,6 +138,7 @@ class DatePicker extends React.Component {
       return;
     }
 
+    this.isDefaultDateAcceptable = true;
```

Comment:
Do we need this?

Reply:
Yes, I want to set this to true when a valid date has been selected.